purchasing_test/unit/conductor/test_costars_upload.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+from purchasing_test.unit.test_base import BaseTestCase
+from purchasing_test.unit.util import (
+    insert_a_role, insert_a_user
+)
+
+class TestCostarsUpload(BaseTestCase):
+    def setUp(self):
+        super(TestCostarsUpload, self).setUp()
+
+        self.conductor_role_id = insert_a_role('conductor')
+        self.conductor = insert_a_user(role=self.conductor_role_id)
+
+        self.staff_role_id = insert_a_role('staff')
+        self.staff = insert_a_user(email='foo2@foo.com', role=self.staff_role_id)
+
+        self.login_user(self.conductor)
+
+    def test_page_locked(self):
+        '''Test page won't render for people without proper roles
+        '''
+        # test that you can't access upload page unless you are signed in with proper role
+        request = self.client.get('/conductor/upload_new')
```

Comment:
Looks like this is failing because you are logged in as `self.conductor` (see line 18) -- you'll need to call `self.logout_user()` to make sure this works as expected.